### PR TITLE
Updated readme

### PR DESCRIPTION
**Pull request by samagragupta**
Fixes : #1

## Did you read the instructions in the [README](https://github.com/the-ethan-hunt/first-timers-guide/blob/master/README.md)?

> Y

## Have you read the instructions in [CONTRIBUTING](https://github.com/the-ethan-hunt/first-timers-guide/blob/master/CONTRIBUTING.md)?

> Y

## Have you written this article by yourself? Have you taken inspiration from anywhere and then written it in your words?

> Written myself

Written index in the README.MD with each topic title in its respective topic and a direct link to the article.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,31 @@
 - [FAQs](#faqs)
 - [License](#license)
 
+### Guides
+
+- [Cloud_development](guides/Cloud_development)
```
Comment (dhruvapte26) [inline README.md]: Write `Cloud Development` instead of `Cloud_development`. Change this for all applicable topics

File: README.md [hunk 2]
```diff
@@ -19,6 +19,31 @@
...
+- [DevOps](guides/DevOps)
+- [Game Development](guides/Game Development)
```
Comment (dhruvapte26) [inline README.md]: Correct this. Another space character should be present to activate the link. Applicable to other non-links also

File: README.md [hunk 3]
```diff
@@ -19,6 +19,31 @@
...
+- [Programming_Paradigm](guides/Programming_Paradigm)
+- [Security](guides/Security)
+- [Starters](guides/Starters)
+- [UI_design](guides/UI_design)
+- [UX_design](guides/UX_design)
+- [Version_Control](guides/Version_Control)
+- [Visual_design](guides/Visual_design)
+- [Web Containers](guides/Web Containers)
+- [Web_Frameworks](guides/Web_Frameworks)
+- [agile](guides/agile)
```
Comment (dhruvapte26) [inline README.md]: Starting letters in uppercase for all

**Review by dhruvapte26 — CHANGES_REQUESTED** [review]

**Comment by dhruvapte26** [comment]
@samagragupta I have requested some changes. Do them and we are good to merge.

**Comment by samagragupta** [comment]
@the-ethan-hunt Done the changes

**Comment by dhruvapte26** [comment]
Merging it

**Pull request merged**